pallets/msa/src/types.rs

```diff
@@ -4,6 +4,8 @@ use scale_info::TypeInfo;
 
 use codec::{Decode, Encode};
 
+pub const EMPTY_FUNCTION: fn(MessageSenderId) -> DispatchResult = |_| Ok(());
```

Comment:
a no-op empty function to pass as a param when there is nothing needs to be done inside creation transaction